media-libs/libass/libass-0.17.3.recipe

```diff
@@ -2,22 +2,32 @@ SUMMARY="Library for SSA/ASS subtitles rendering"
 DESCRIPTION="Libass is a portable subtitle renderer for the ASS/SSA (Advanced \
 Substation Alpha/Substation Alpha) subtitle format."
 HOMEPAGE="https://github.com/libass/libass"
-COPYRIGHT="2006-2021 libass contributors
+COPYRIGHT="2006-2022 libass contributors
 	2006 Evgeniy Stepanov
 	2011-2014 Yu Zhuohuang
-	2013 Rodger Combs
-	2015 Grigori Goronzy
-	2016 Oleg Oshmyan
-	2016 Vabishchevich Nikolay"
+	2013-2021 rcombs
+	2009-2015 Grigori Goronzy
+	2015-2016 Oleg Oshmyan
+	2014-2017 Vabishchevich Nikolay
+	2005-2024 x264 project
+	2013 Stefano Pigozzi
+	2015 Stephan Vedder
+	1994 Sun Microsystems Inc
+	1988-1993 The Regents of the University of California
+	2015-2018 Janne Grunau
+	2015 Martin Storsjo
+	2018 Two Orioles LLC
+	2018 VideoLAN and dav1d authors
+	2021 王一 Wang Yi"
 LICENSE="ISC"
-REVISION="1"
+REVISION="2"
```

Comment:
REVISION should stay as 1. It is supposed to be increased when the recipe is updated for the same version as before so that the package is rebuilt and available as an update. When upgrading to a new version, it is reset to 1.

Reply:
fixed